vesc_hw_interface/include/vesc_hw_interface/vesc_wheel_controller.h

```diff
@@ -59,13 +59,15 @@ class VescWheelController
 
   double velocity_reference_;
   double steps_;
+  double prev_steps_;
```

Comment:
I think these two parameters need to be ints or the difference will be wrong on overflow.

Reply:
I think that we should call `control()` with relative `steps` too, if we consider overflow.

Reply:
it's true

Reply:
> I think these two parameters need to be ints or the difference will be wrong on overflow.

I'm going to need an explanation on this one. I understand that overflow will eventually happen but what I don't understand is how casting the variables to ints will solve this problem.

Reply:
If the vesc's tachometer overflow max value of `int`, `prev_steps` is positive large value and `steps` is negative large value.
If so, `double - double` calculation such as `steps_ - prev_steps_` and `target_steps_ - current_steps_` can't server overflow, because `double` can handle larger values than `int`.

Reply:
>> I think these two parameters need to be ints or the difference will be wrong on overflow.
> If the vesc's tachometer overflow max value of `int`, `prev_steps` is positive large value and `steps` is negative large value. If so, `double - double` calculation such as `steps_ - prev_steps_` and `target_steps_ - current_steps_` can't detect overflow, because `double` can handle larger values than `int`.

Thanks for the clarification, will change it accordingly.